docs/source/kedro_project_setup/settings.md

```diff
@@ -0,0 +1,98 @@
+# Project settings
+
+A Kedro project's `settings.py` file contains the settings for the project, such as library component registration, custom hooks registration, etc. This document explains how settings work and which settings are available.
+
+## Default settings
+A Kedro settings file doesn’t have to define any settings if it doesn’t need to. Each setting has a sensible default value. These defaults live in the module `src/<project_name>/settings.py`.
+
+## Available settings
+
+#### HOOKS
+Default: `tuple()`
+
+Instantiate and list your project hooks.
```

Comment:
Could we link this to the docs page on hooks?

Reply:
Done